bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/ColumnDescriptorAdapter.java

```diff
@@ -194,8 +194,9 @@ public static String buildGarbageCollectionExpression(HColumnDescriptor columnDe
   private static void convertGarbageCollectionExpression(String gcExpression,
       HColumnDescriptor columnDescriptor) {
     if (Strings.isNullOrEmpty(gcExpression)) {
-      // No GC Expression means keep all versions, MaxVersions = 0.
-      columnDescriptor.setMaxVersions(0);
+      // No GC Expression means keep all versions. HBase doesn't support this, but this is the
+      // default Bigtable behavior. 
```

Comment:
The default behavior of hbase is 1 version.  The default behavior of bigtable is infinite versions.  If we don't set anything, the user will see a maximum of 1 version, when the reality is infinite versions.